include/circt/Dialect/OM/OMAttributes.td

```diff
@@ -49,6 +49,10 @@ def OMSymbolRefAttr : AttrDef<OMDialect, "SymbolRef", [TypedAttrInterface]> {
 
   let extraClassDeclaration = [{
     mlir::Type getType();
+    // Get the SymbolRefAttr to the symbol represented by this operation.
+    static om::SymbolRefAttr get(mlir::Operation *op);
```

Comment:
Any reason why these were defined in C++? I think it's worth being consistent one way or the other.

Reply:
Thanks for the suggestion, I updated and moved all the defs to the C++.